cli/lib/kontena/plugin_manager.rb

```diff
@@ -174,10 +174,9 @@ def load_plugins
                 $stderr.puts " [#{Kontena.pastel.red('error')}] Plugin #{Kontena.pastel.cyan(plugin_name)} (#{spec.version}) is not compatible with the current cli version."
                 $stderr.puts "         To update the plugin, run 'kontena plugin install #{plugin_name}'"
               end
-            rescue LoadError => ex
-              $stderr.puts " [#{Kontena.pastel.red('error')}] Failed to load plugin: #{spec.name}"
+            rescue Exception => ex
```

Comment:
LoadError is not from StandardError. Ctrl-c will abort plugin loading but continue kontenaing.

```ruby
irb(main):001:0> begin
irb(main):002:1*     raise LoadError
irb(main):003:1>   rescue => ex
irb(main):004:1>   puts ex.inspect
irb(main):005:1>   end
LoadError: LoadError
        from (irb):2

irb(main):006:0> begin
irb(main):007:1*     raise LoadError
irb(main):008:1>   rescue Exception => ex
irb(main):009:1>   puts ex.inspect
irb(main):010:1>   end
#<LoadError: LoadError>
```